src/Manager/BugManager.php

```diff
@@ -38,10 +48,23 @@ public function convertToFeature(Bug $bug): Feature
     {
         $feature = UserRequestMapper::mapBugToFeature($bug);
         $bug->resolve();
+        $this->bugLifecycleStateMachine->apply($bug, 'dismiss');
 
         $this->em->persist($feature);
         $this->em->flush();
 
         return $feature;
     }
+
+    public function solve(Bug $bug): void
+    {
+        $this->bugLifecycleStateMachine->apply($bug, 'solve');
+        $this->markDone($bug);
+    }
+
+    public function dismiss(Bug $bug): void
+    {
+        $this->bugLifecycleStateMachine->apply($bug, 'dismiss');
+        $this->markDone($bug);
+    }
```

Comment:
I wonder if makeDone() should be called by a Listener or something, because we always want à solved bug to be marked done (same for dismiss)